_data/simple-icons.json

```
@@ -6927,6 +6927,11 @@
             "hex": "199900",
             "source": "https://github.com/Leaflet/Leaflet/blob/d843c3b88486713827d7e860b58bdba75bfbd5a2/src/images/logo.svg"
         },
+        {
+            "title": "League of Legends",
+            "hex": "BF913B",
```

```suggestion
            "hex": "C28F2C",
```

Taken from the source icon on the homepage.